functions/helloworld/scala-helloworld/pom.xml

```
@@ -43,8 +48,8 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
       <version>2.13.2</version>
-    </dependency>
-
+    </dependency>=
```

I think this is your test failure
```suggestion
    </dependency>
```

---

Oops - thanks!